build .rpm and .deb packages to automatically install and configure frankenphp (#1497)
dunglas/frankenphp: docs/config.md

FrankenPHP, Caddy as well as the Mercure and Vulcain modules can be configured using [the formats supported by Caddy](https://caddyserver.com/docs/getting-started#your-first-config).

In [the Docker images](docker.md), the `Caddyfile` is located at `/etc/frankenphp/Caddyfile`.
The static binary will also look for the `Caddyfile` in the directory where the `frankenphp run` command is executed.
You can specify a custom path with the `-c` or `--config` option.

PHP itself can be configured [using a `php.ini` file](https://www.php.net/manual/en/configuration.file.php).

> The PHP interpreter will look in the following locations:

Reviewer: ```suggestion
Depending on your installation method, the PHP interpreter will look for configuration files in locations described above.
```

> The PHP interpreter will look in the following locations:

Reviewer: ```suggestion
Depending on your installation method, the PHP interpreter will look for configuration files in locations described above.
```

> Docker:

Reviewer: ```suggestion
## Docker
```

> - php.ini: `/usr/local/etc/php/php.ini` No php.ini is provided by default.

Reviewer: ```suggestion
- `php.ini`: `/usr/local/etc/php/php.ini` (no `php.ini` is provided by default)
```

- additional configuration files: `/usr/local/etc/php/conf.d/*.ini`

> - php extensions: `/usr/local/lib/php/extensions/no-debug-zts-<YYYYMMDD>/`

Reviewer: ```suggestion
- PHP extensions: `/usr/local/lib/php/extensions/no-debug-zts-<YYYYMMDD>/`
```

RUN cp $PHP_INI_DIR/php.ini-development $PHP_INI_DIR/php.ini
```

> dev.Dockerfile:

Reviewer: You can drop this section, or move it to `CONTRIBUTING.md`, `dev.Dockerfile` is only for contributors to FrankenPHP itself.

- php.ini: `/etc/frankenphp/php.ini` A php.ini file with development presets is provided by default.
- additional configuration files: `/etc/frankenphp/php.d/*.ini`
- php extensions: `/usr/lib/frankenphp/modules/`

> FrankenPHP installation (.rpm or .deb):

Reviewer: ```suggestion
## RPM and Debian packages
```

> - php.ini: `/etc/frankenphp/php.ini` A php.ini file with production presets is provided by default.

Reviewer: ```suggestion
- `php.ini`: `/etc/frankenphp/php.ini` (a `php.ini` file with production presets is provided by default)
```

- additional configuration files: `/etc/frankenphp/php.d/*.ini`

> - php extensions: `/usr/lib/frankenphp/modules/`

Reviewer: ```suggestion
- PHP extensions: `/usr/lib/frankenphp/modules/`
```

> Static binary:

Reviewer: ```suggestion
## Static binary
```

> - php.ini: The directory in which `frankenphp run` or `frankenphp php-server` is executed, then `/etc/frankenphp/php.ini`

Reviewer: ```suggestion
- `php.ini`: The directory in which `frankenphp run` or `frankenphp php-server` is executed, then `/etc/frankenphp/php.ini`
```

- additional configuration files: `/etc/frankenphp/php.d/*.ini`

> - php extensions: cannot be loaded

Reviewer: ```suggestion
- PHP extensions: cannot be loaded, bundle them in the binary itself
```
